tests/test_paperscraper.py

```diff
@@ -197,7 +197,7 @@ async def test_google_search_papers(self) -> None:
 
     async def test_high_limit(self) -> None:
         papers = await paperscraper.a_search_papers(
-            "molecular dynamics", search_type="google", year="2019-2023", limit=45
```

Comment:
Just to share, the reason this test had `limit=45` was to make it be `> 2 * 20` as regression protection for https://github.com/blackadad/paper-scraper/pull/53

I guess 25 still works, as it's `> 1 * 20`

Maybe we should alternately increase the timeout used by `pytest-timeout`: `timeout = 300` sec --> `timeout = 450` sec. I had picked 300 originally because I felt 5 mins for one scrape was already too much